model/Licensing/Licensing.md

```diff
@@ -0,0 +1,97 @@
+SPDX-License-Identifier: Community-Spec-1.0
+
+# Licensing
+
+## Summary
+
+The Licensing Profile defines a minimum set of license information required to comply with typical license compliance use cases.
+
+## Description
+
+The Licensing profile only contains additional restrictions.
+Classes and Property restrictions are defined in the SimpleLicensingProfile
+(Classes and Properties associated with string license expressions) and in the ExpandedLicensingProfile (Classes and Properties used for a
+fully parsed syntax tree of license expressions).
+
+There are 2 relationships which are required for any Software Artifact - declaredLicense and concludedLicense.
+
+A declaredLicense is the license identified in text in the software package,
```

Comment:
I think this is missing a pretty key piece from the "summary" part of the prior text, which isn't there now but is visible in the PR that removed it, at https://github.com/spdx/spdx-3-model/pull/448/files#diff-7c90a0506e9d8122276df9c2e1d92a0115bc693c4f2cc53c95fab28c7be943f1L7-L8:

A declaredLicense "[i]dentifies the license information actually found in the software Package, File or Snippet, for example as detected by use of automated tooling."

The "as detected by use of automated tooling" is pretty important to make it clear that this is where information about detected licenses should be expressed.

Reply:
Updated per suggestion.